content/docs/querying/functions.md

```diff
@@ -80,6 +86,54 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the
```

Comment:
Given that we had this discussion last week (and no clear outcome), I think it's ok to keep it as it is for now.